telco-hub/configuration/reference-crs/optional/cert-manager/certManagerSubscription.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+  labels:
+    operators.coreos.com/openshift-cert-manager-operator.cert-manager-operator: ''
+spec:
+  channel: stable-v1
+  name: openshift-cert-manager-operator
+  source: redhat-operators
```

Comment:
Other operators in the reference use "redhat-operators-disconnected". For testing on your cluster you can replace it but for consistency here we should match with others.
```suggestion
  source: redhat-operators-disconnected
```